src/main/java/edu/greenblitz/tobyDetermined/subsystems/DriveTrain/IO/GyroIOs/SimGyro.java

```diff
@@ -0,0 +1,15 @@
+package edu.greenblitz.tobyDetermined.subsystems.DriveTrain.IO.GyroIOs;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class SimGyro implements GyroIO{
```

Comment:
Use a full indicative name (don't be afraid to write Simulation), someone might think that it is a gyro related to those people:
![image](https://github.com/GreenBlitz/Charrrged-Up-2023/assets/26098399/7c72ae76-f287-44e0-be46-ce6d685ae6a1)

Reply:
lol